bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -1359,7 +1359,6 @@ public void updateProperties(ShellyDeviceProfile profile, ShellySettingsStatus s
         properties.put(PROPERTY_SERVICE_NAME, config.serviceName);
         String deviceName = getString(profile.settings.name);
         properties.put(PROPERTY_SERVICE_NAME, config.serviceName);
-        properties.put(PROPERTY_DEV_GEN, !profile.isGen2 ? "1" : "2");
```

Comment:
Just to be sure, is this property removed on purpose?

Unrelated to this PR, just observation: The three lines above seems suspicious. `PROPERTY_SERVICE_NAME` is set twice.

Reply:
yes, and 1st PROPERTY_SERVICE_NAME removed
thx

Reply:
@jlaur changes are applied